try {
	const foo = bar();
} <mark>catch {
    console.err('Oops');
}</mark>

try {
	const foo = bar();
} <mark>catch (e) {
	console.log(e);
}</mark> finally{
    console.log('end');
}